src/main/resources/data.sql

```diff
@@ -1,15 +1,18 @@
-CREATE TABLE RACING_GAME (
-    id          INT         NOT NULL AUTO_INCREMENT,
-    count       INT         NOT NULL,
-    created_at  DATETIME    NOT NULL default current_timestamp,
+CREATE TABLE RACING_GAME
+(
+    id         INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
```

Comment:
PK를 두 번 선언하고 있네요!
(위에서 속성으로, 아래에서 별도로)

---
```suggestion
    id         INT UNSIGNED NOT NULL AUTO_INCREMENT,
```
이런식으로 선언하면 물론 H2에서는 UNSIGNED가 동작하지 않겠지만, MySQL로 바꿨을 때 바로 적용이 되겠죠 😉 
(H2는 실제 DB로는 거의 사용하지 않으니까요)

product는 문제없이 동작하는데, test에서 오류가 발생하는군요!
`@JdbcTest`와 `h2의 UNSIGNED`와 궁합이 잘 맞지 않는 듯 하네요 😢 

아래와 같이 설정하면 오류가 발생하지 않지만,
별도로 수정하진 않고 `UNSIGNED`에 대해 학습한 것에 의의를 둬도 좋을 것 같아요. 😃 

``` diff
+ @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+ @Transactional
- @JdbcTest
```